internal/driver/capabilities.go

```diff
@@ -8,7 +8,6 @@ func ControllerServiceCapabilities() []*csi.ControllerServiceCapability {
 	capabilities := []csi.ControllerServiceCapability_RPC_Type{
 		csi.ControllerServiceCapability_RPC_CREATE_DELETE_VOLUME,
 		csi.ControllerServiceCapability_RPC_PUBLISH_UNPUBLISH_VOLUME,
-		csi.ControllerServiceCapability_RPC_PUBLISH_READONLY,
```

Comment:
Do we need to remove this capability? Or we are removing this because we don't support read only volumes?

Reply:
Just read the PR description and figured out that we are removing this as we currently don't support it.